Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -541,12 +541,10 @@ export default function widgetBehavior(publicAPI, model) {
       model.isDragging = true;
       model.apiSpecificRenderWindow.setCursor('grabbing');
       model.interactor.requestAnimation(publicAPI);
-      publicAPI.invokeStartInteractionEvent();
-
-      return macro.EVENT_ABORT;
     }
 
-    return macro.VOID;
+    publicAPI.invokeStartInteractionEvent();
+    return macro.EVENT_ABORT;
```

Comment:
it seems very fragile. 
How about a unit test ? (that tests that a click on text handle "eats" mouse press AND starts interaction event.

Reply:
@tomsuchel WDYT ?